.github/workflows/test-application.yaml

```diff
@@ -20,35 +20,47 @@ jobs:
                       lint: false
                       symfony-version: '^2.8'
                       elasticsearch-package-constraint: '^5.0'
+                      elasticsearch-version: '5.6.14'
 
                     - php-version: '7.0'
                       lint: false
                       symfony-version: '^3.4'
                       elasticsearch-package-constraint: '^5.0'
+                      elasticsearch-version: '5.6.14'
 
                     - php-version: '7.2'
                       lint: false
                       symfony-version: '^3.4'
                       elasticsearch-package-constraint: '^5.0'
+                      elasticsearch-version: '5.6.14'
 
                     - php-version: '7.3'
                       lint: false
                       symfony-version: '^4.4'
                       elasticsearch-package-constraint: '^5.0'
+                      elasticsearch-version: '5.6.14'
 
                     - php-version: '7.4'
                       lint: true
                       symfony-version: '^5.0'
                       elasticsearch-package-constraint: '^5.0'
+                      elasticsearch-version: '5.6.14'
+
+                    - php-version: '8.0'
```

Comment:
can you add a additional test case with `php-version` `7.4` and `elasticsearch-version`: `7.11.11` I think currently there are some elasticsearch 5 specific tests which will fail when using es7.

Reply:
That's a good idea. I am trying to fix the failing test and will do that after.